### PR TITLE
Re-add the --wait flag to the helm3 installation

### PR DESCRIPTION
**Pull request by Waterdrips**
## Description
This commit re-adds the wait flag to helm3 installation
and sets the default --version to empty string, this
installs the latest chart version by default. When we need
a specific version we can set it (see cert-manager)

This was removed in a previous commit was it was not being used 
but it should be used. It provides a better UX, as apps are ready once
k3sup returns. Otherwise when chaining installs like cert-manager and
openfaas ingress (which relies on the cert-manager installation being ready)
we an get errors.

I have also added a const to represent default chart version rather then using the 
empty string.

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))
Reverting a removed feature and enabling it

## How Has This Been Tested?

Installed a chart with version (cert-manager) with helm2 & helm3, validated
that the install was successful and the correct version was installed

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: cmd/apps/certmanager_app.go [hunk 1]
```diff
@@ -120,7 +121,7 @@ func MakeInstallCertManager() *cobra.Command {
 				return err
 			}
 		} else {
-			err = templateChart(chartPath, "cert-manager", namespace, outputPath, "values.yaml", "v0.12.0", nil)
+			err = templateChart(chartPath, "cert-manager", namespace, outputPath, "values.yaml", nil)
```
Comment (alexellis) [inline cmd/apps/certmanager_app.go]: I think this reverted the version or pinning

Comment (Waterdrips) [reply cmd/apps/certmanager_app.go]: its moved into fetchChart (for helm2)

**Comment by Waterdrips** [comment]
> In this PR can we also add --wait as a flag for each that's consumed?

Is there any reason not to use --wait for all of the commands?

**Comment by alexellis** [comment]
Yes it's blocking and slow. Using async means you can pull images in parallel.

**Pull request merged**